overlord/snapstate/snapstate.go

```diff
@@ -720,6 +729,12 @@ func Update(st *state.State, name, channel string, revision snap.Revision, userI
 		return nil, err
 	}
 	if len(tts) == 0 && len(updates) == 0 {
```

Comment:
I fear this is a bit too simple, there may be no revision update even with len(tts) != 0 (just alias changes from assertions) or even with len(updates) != 0, as I said in the standup we need to review the return of UpdateMany/doUpdate to be clearer